packages/mitex/specs/latex/standard.typ

```diff
@@ -146,7 +146,7 @@
   textstyle: greedy-handle("mitexinline", math.inline),
   scriptstyle: greedy-handle("mitexscript", math.script),
   scriptscriptstyle: greedy-handle("mitexsscript", math.sscript),
-  bf: greedy-handle("mitexbold", math.bold),
+  bf: define-cmd(1, alias: "mitexbold", handle: it => math.bold(math.upright(it))),
```

Comment:
Thanks. Can't we continue to use `greedy-handle`? Does `greedy-handle("mitexbf", it = math.bold(math.upright(it)))` work?

Reply:
`greedy-handle("mitexbold", it => math.bold(math.upright(it)))`  can also work after a quick try.

Reply:
I will change this commit before CI is fixed.